CONTRIBUTING.md

```diff
@@ -26,6 +26,14 @@ Every integration test should have `integration` mark. By default, integration t
 
 To execute only integration tests run `pytest -m integration`.
 
+### Installation of external plugins used in integration tests
+
+To run integration tests you have to install external plugins which are part of the repository:
+```bash
+pip install -e test_external_plugins/snowpark_hello_single_command
+pip install -e test_external_plugins/multilingual_hello_command_group
+```
```

Comment:
It sounds like these 2 plugins are required for integration tests? Are they?

Reply:
They are required (as we discussed today).